ccpp/data/GFS_typedefs.meta

```diff
@@ -8759,6 +8773,33 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[ltg1_max]
+  standard_name = gsl_lightning_threat_index_1
```

Comment:
I would remove "gsl" from this unless/until there are multiple lightning threat indices to differentiate. Same with ...2,3.

Reply:
I was matching the names in UPP, but I'm happy to remove them here.

Reply:
The "gsl" parts of the names are gone.